libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCredentials.m

```diff
@@ -97,17 +97,18 @@ - (id)initWithCoder:(NSCoder *)coder {
             _encrypted = (encryptedBool
                           ? [encryptedBool boolValue]
                           : [coder decodeBoolForKey:@"SFOAuthEncrypted"]);
-            if ([self isMemberOfClass:[SFOAuthCredentials class]]) {
```

Comment:
That check is [ancient](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/3e48d5b5e1f336d91fbda99da36dddc4c152a8d2/libs/SalesforceOAuth/SalesforceOAuth/Classes/SFOAuthCredentials.m) and when introduced it was only to guard the setting of refresh and access token - maybe we had a different credentials class early on??